src/serialize_sequence.cc

```diff
@@ -182,4 +182,18 @@ void store_sequence(const std::filesystem::path& folder, const Sequence& seq)
         store_step(seq_path / extract_filename_step(++idx, max_digits, step), step);
 }
 
+
+void remove_sequence(const std::filesystem::path& seq_path)
+{
+    try
+    {
+        if (std::filesystem::exists(seq_path))
+            std::filesystem::remove_all(seq_path);
+    }
+    catch(const std::exception& e)
+    {
+        auto err = errno;
+        throw Error(gul14::cat("I/O error: ", e.what(), ", error=", std::strerror(err)));
```

Comment:
According to [cppreference](https://en.cppreference.com/w/cpp/filesystem/remove), you have the choice between two versions of the `std::filesystem` functions: One throws [filesystem::filesystem_error](https://en.cppreference.com/w/cpp/filesystem/filesystem_error) exceptions, the other one returns an error code by reference. Using `errno` might work, but is not guaranteed by the standard.

Reply:
Equivalent to function store_sequence (one above)

Reply:
Good point. It is still (potentially) wrong, so we should fix it also in `store_sequence()`.

Reply:
Let's do it this way: Either we remove the `errno` code in this pull request (also from the old code), or we create an issue to remember doing that later on. Pick one. :)

Reply:
Opened #81, closing this thread.